components/automatic_points_table/commons/automatic_points_table.lua

```diff
@@ -0,0 +1,103 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:AutomaticPointsTable
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+---- This module creates a Points Table that automatically gets data from the prizepools
+---- of set tournaments
+
+local Class = require('Module:Class')
+-- local Table = require('Module:Table')

```

Comment:
Since Table is used now we can uncomment it, it was commented to pass the automatic checks and would've been uncommented later when its usage is added back.
```suggestion
local Table = require('Module:Table')

```